bedrock/firefox/templates/firefox/new.html

```diff
@@ -150,9 +158,36 @@ <h1 class="large" id="main-headline">
       <div id="newsletter-latest">
         {{ email_newsletter_form() }}
       </div>
-    </div>{#-- /#download-ready --#}
+    </div>{#-- /#scene1-main --#}
+
+    <div id="scene2-main" class="scene-content scene2 hidden">
+    {% if l10n_has_tag('firefox_new_mobile_promo') and waffle.switch('firefox-ios-global') %}
+      <p class="help">
+        {{_('Your download should begin automatically. If not, <a id="%s" href="">click here</a>. It could take a few minutes, but it’s worth the wait.')|format('direct-download-link')}}
+        <a rel="external" href="https://support.mozilla.org/products/firefox?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=need-help-link">{{_('Need help?')}}</a>
+      </p>
+
+      <div id="thankyou" tabindex="0">
+        {{ _('While it installs, add it to your Android and iOS devices.') }}
+      </div>
 
-    <div id="download-started" class="download-content hidden">
+      <div class="mobile-download-buttons">
+        <ul>
+          <li class="android">
+            <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
```

Comment:
Not introduced by this PR but, just noticed that this link reads:
`https://play.google.com/store/apps/details?id=org.mozilla.firefox&referrer=utm_source%3Dmozilla%26utm_medium%3DReferral%26utm_campaign%3Dmozilla-org`

Note the `referrer=utm_source....`

Should it not be `referrer=&amp;utm_source...`